conf/ini-files/homo_sapiens_gca018466835v1.ini

```diff
@@ -47,26 +47,17 @@ SPECIES_RELEASE_VERSION = 1
 ####################
 
 [ENSEMBL_EXTERNAL_URLS]
-# Accept defaults
-
-####################
-# Configure search example links
-####################
-
-
+CLIN_SIG = http://www.ncbi.nlm.nih.gov/clinvar/docs/clinsig/
```

Comment:
For future reference, it's better to put these into DEFAULTS.ini if they're going to be shared among species, otherwise it's a pain to update them all if they change.